dev/bin/manager-run.sh

```diff
@@ -5,12 +5,13 @@
 #
 # Env vars:
 # - MANAGED_CONNECTORS_CLUSTER_ID: cluster where managed connectors will be deployed (required only if MC actions are used, default="empty")
-# - MANAGED_KAFKA_INSTANCE_NAME: set the managed kafka instance name (required)
 # - OPENSHIFT_OFFLINE_TOKEN: Red Hat account offline token (required, get it at https://console.redhat.com/openshift/token)
 ########
 
 SCRIPT_DIR_PATH=`dirname "${BASH_SOURCE[0]}"`
 
+export MANAGED_KAFKA_INSTANCE_NAME=rhose-local-development
```

Comment:
I think we should centralise it in some common param file but it is probably better to do it in the other PR with the refactoring of the scripts https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/423

Reply:
Can this not written to `local_env` like we do for `MANAGER_URL` etc?

(or, following @danielezonca comment, move them all to somewhere central.. but that's a different JIRA/PR)

Reply:
e.g. `LOCAL_ENV_FILE` is read from rather than written to!?

Reply:
created https://issues.redhat.com/browse/MGDOBR-502